### ISSUE TITLE
Building problem: CPL parts Q1 and Q2 90 degrees out of alignment with trace contacts

### ISSUE THREAD

**Issue by snetsplit** [dudanov/iot-uni-dongle#12]
Two of the parts, Q1 and Q2 are 90 degrees rotated out of alignment with the contacts. I had to modify the cpl.csv to rotate them, but there's two possible rotations. There's a purple dot in JLCPCB order tool but I'm not sure which side it goes on. From the animation it looks like the label text should go down to up. So I hope I'm correct, but I don't know.

Can someone help

How the CPL file is off here
![default](https://user-images.githubusercontent.com/15273357/180870601-81c0f11c-6344-4678-8f29-413780bf8860.png)

Is this how it should be?
![rotated Q1 and Q2](https://user-images.githubusercontent.com/15273357/180870596-9ca3b444-8165-4d09-b50a-78cb34deb2bc.png)

**Comment by malves61** [comment]
I have ordered from jlcpcb a few months ago, 5 boards were manufactured as your 'best guess' without any modification. For what is worth I have not yet finished assembling them (ESP8266 and USB connector) or tested them - you know 'projects' piled over it.

**Comment by snetsplit** [comment]
Gonna just hope and order. If it's wrong should be able to resolder it.

**Comment by ozumado** [comment]
Hi, did you find out whats the right direction?

**Comment by snetsplit** [comment]
Nope, and won't for a bit. Unexpected vet bill and messed up my finances for a bit

**Comment by ozumado** [comment]
So I just came with a brilliant idea. I have purchased the iot-uni-dongle last year and its in use at my parents house. I will take high resolution/quality pictures so it will be easier to verify the correct orientation.

**Comment by snetsplit** [comment]
That would be awesome!

**Comment by dudanov** [comment]
Guys, the rotation angles of the components are not standardized. Whichever angle you specify, there is no guarantee that it will be correct. Usually, in PCB assembly services, this stage is manually pre-moderated by engineers. In JLCPCB so, in PCBWAY too. As a general rule, it is recommended to use a silkscreen that identifies the first pin of a component. Engineers basically rely on it.

Don't waste your time looking for the right rotation angle. Trust PCB assembly service engineers. 😉

**Comment by snetsplit** [comment]
Very good to know! Thank you!
